src/NimBLEClient.cpp

```diff
@@ -436,6 +436,20 @@ void NimBLEClient::updateConnParams(uint16_t minInterval, uint16_t maxInterval,
 } // updateConnParams
 
 
+/**
+ * @brief Request an update of the data packet length.
+ * * Can only be used after a connection has been established.
+ * @param [in] tx_octets The preferred number of payload octets to use (Range 0x001B-0x00FB).
+ * @param [in] tx_time The preferred number of microseconds to use when transmitting a single packet (Range 0x0148-0x4290).
+ */
+void NimBLEServer::setDataLen(uint16_t tx_octets, uint16_t tx_time) {
```

Comment:
This needs to be `NimBLEClient::setDataLen` here.

Reply:
Whops, done.